### PR TITLE
adds a space between 'go' and 'to' on line 15 and and delete the void…

### PR DESCRIPTION
**Pull request by MikeGalyen**
… modifier on lines 64, 65, and 66

### CODE REVIEW

**Comment by kang8** [comment]
Looks so good, Why closed it.

**Comment by MikeGalyen** [comment]
> Looks so good, Why closed it.

I'm new to contributing on GitHub. My mistake.

File: languages/java.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ for(dataType item : array) {
     do{  
       System.out.println(i);  
       i++;
```
Comment (kang8) [inline languages/java.md:247]: There are also end-of-line spaces.

File: languages/java.md [hunk 2]
```diff
@@ -189,15 +189,15 @@ int c = a + b;
   {
     //Statement
   }
-``` 
+```
 **Example:**
 ```java
```
Comment (kang8) [inline languages/java.md:194]: There are also end-of-line spaces.

**Review by kang8 — CHANGES_REQUESTED** [review]

**Comment by LeCoupa** [comment]
Thank you to both of you!

**Pull request merged**